tests/test_views.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from django.test import override_settings
+from atol.models import Receipt, ReceiptStatus
+
+
+@pytest.mark.django_db(transaction=True)
```

Comment:
Можно проще, в модуле написать:

```
import pytest
pytestmark = pytest.mark.django_db(transaction=True)
```

>in which case it will be applied to all functions and methods defined in the module.